_posts/2022-08-29-changelog-177.markdown

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #177"
+date: 2022-08-29 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#8937] Enable `Unused imports` inspection if proc macros are enabled. Proc macros can be enabled using `org.rust.cargo.evaluate.build.scripts` and `org.rust.macros.proc` [experimental features](https://plugins.jetbrains.com/plugin/8182-rust/docs/rust-faq.html#experimental-features)
```

Comment:
Let's move this item after `Debugger pretty-printers improvements` item since it describes changes only for some people